eth2.0-implementers-calls/call_021.md

```diff
@@ -0,0 +1,698 @@
+* And again, you don't want to first only run pre-states, it's more complicated than that.
+
+**Justin Drake**:
+* Yeah it'd definitely coverage driven and there's the randomization aspect.
+
+**Danny Ryan**:
+* Cool, any other questions on this before we move in?
+* Preston has a question: 
+    * "How does the Fuzz harness look: is it libfuzz or is this new tool compatible?"
+
+**Preston**
+* Hey, I think Proto (Diederik Loerakker) just answered that he said that it would be using libfuzz to, basically, you give the signal feedback: did you get farther through the test or was this the worst fuzzing scenario?
+* So that answers the question, thanks.
+
+**Diederik Loerakker:**
+* Some people here are already libfuzz's libfuzzer
+* We have Peter working on this with lots of experience with it and extending it
+* What I was just saying about these 2 different kinds of states we're working on. We have the Beacon States and the Block
+* We don't want to involve only the block inputs because if you're always Fuzzing the same states, you don't really get the same coverage as you could if you're Fuzzing multiple states
+* So this why we're trying to improve on the normal architecture
+* Yet it's somewhat similar to libfuzz service since this is what's being used internally
+
+**Justin Drake**:
+* Did he suggest that the current approach was good enough at exploring many different paths and finding bugs. 
+* I think the main limit was the speed of PyStack, that you couldn't Fuzz that much, it would be rather slow.
+* Hopefully once we do differential Fuzzing on the Lighthouse, we'll get a bit more confidence.
+
+**Diederik Loerakker:**
+* We get a lot of poor performance because of spec and hopefully also with clients. 
+* I think with PySpec is that there are these functions that are not really pleasant outs they all inefficiently repeat themselves, not precomputing anything
+* Now in the next Go spec updates, I optimize, precompute this, precompute that, so we get half, almost client speed spec.
+* I tried to keep this really portable.
+
+**Danny Ryan**:
+* One of the paths might be to differentially Fuzz against the Go spec and the clients and then if we find issues, to then go back to the Py spec and make sure we're in agreement on what the functionality should be.
+* Okay, let's move on Fuzzing, thank you. 
+* We'll keep y'all updated over the coming weeks on that.
+
+# 2. Client Updates
+[Timestamp 13:50](https://youtu.be/YB8o_5qjNBc?t=830)
+
+## Nimbus
+**Mamy**:
+* We launched our testnet 1, based on libp2p daemon, this morning
+* We need to adapt the script, so that it starts with a single common because libp2p daemon is not built but it's almost done. Pretty soon by the end of the week.
+* We are towards v0.8, all the small changes were integrated and right now a big focus is SSE
```

Comment:
It's SSZ, not SSE :) 

```suggestion
* We are towards v0.8, all the small changes were integrated and right now a big focus is SSZ
```

Reply:
Thank you, Terence! I need to read through all the specs myself, haha